blogs/data-mover/data-mover-1.3/data-mover-1.3.0-overview.md

```diff
@@ -0,0 +1,230 @@
+</tr>
+<tr>
+<td>Prepared</td>
+<td>The snapshot has been exposed, the related controller is starting to process the upload</td>
+</tr>
+<tr>
+<td>InProgress</td>
+<td>The data upload is in progress</td>
+</tr>
+<tr>
+<td>Canceling</td>
+<td>The data upload is being canceled</td>
+</tr>
+<tr>
+<td>Canceled</td>
+<td>The data upload has been canceled</td>
+</tr>
+<tr>
+<td>Completed</td>
+<td>The data upload has completed</td>
+</tr>
+<tr>
+<td>Failed</td>
+<td>The data upload has failed</td>
+</tr>
+</tbody>
+</table>
+
+<h2>Backup Process</h2>
+<div>
+A user creates a backup CR with the snapshotMoveData option set to true. Velero calls the BIA V2 api to create a CSI VolumeSnapshot request. The status will move from `New` to `InProgress`.<br><br>
+
+After the VolumeSnapshots are created, you will see one or more DataUpload CRs created.  You may also see some temporary objects (i.e., pods, PVCs, PVs) created in protected (openshift-adp) namespace. The temporary objects are created to assist in the data movement. The status of the DataUpload object will progress from `New` to `Accepted` to `InProgress`. <br>
+
+Now working from the Data Mover plugin the CSI snapshot is mounted from the Node-Agent. The DataUpload Controller then works with Kopia ( the uploader ) to move the object off cluster to the Unified Repo Backup repository off cluster.  The status is once again reconciled and the backup CR is moved to complete.<br>
+
+Users can see the DataUpload objects move to a terminal status of either `Completed`, `Failed` or `Canceled`
+Once the object has been uploaded any intermediate objects like the VolumeSnapshot and VolumeSnapshotContents will be removed. Finally the backup object status will be updated with it's terminal status.
+
+</div>
+<hr>
+
+<p dir="auto"><img alt="backup-13-workflow" src="backup-13-workflow.png" width="850" /></p>
+
+<div>
+<hr>
+A more in depth visualization of the backup workflow with Data Mover is found below.
+<hr>
+</div>
+
+<p dir="auto"><img alt="data-mover-13-backup-sequence" src="data-mover-13-backup-sequence.png" width="850" /></p>
+
+<h2>Restore Process</h2>
+<div>
+A user creates a restore CR, no additional data mover options or parameters are required. Velero calls the RIA V2 api to create a PV and PVC in the protected namespace (openshift-adp). The backup status will move from `New` to `InProgress`.<br><br>
+
+The data from backup is queried from the remotely stored DataUpload CR and written to the local ConfigMaps.  A ConfigMap is created for each PV to be restored. These Configmaps are temporary objects that are deleted upon the restore's workflow completion. The ConfigMap stores vital information like the Repo Snapshot ID or VolumeSnapshotContent name.  The data stored in the ConfigMap is then used to build the DataDownload CR spec. 
+
+The DataMover plugin creates the DataDownload CR and DataDownload Controller reconciles on the CR.  The Node-Agent begins the download the backed up PV data from S3. 
```

Comment:
NIT: download ^^`of`^^ the backed ...

Reply:
The CSI plugin creates the DataDownload CR and DataDownload Controller reconciles on the CR.  The Node-Agent begins the download of the backed up PV data from S3.